src/test/java/com/amazon/jenkins/ec2fleet/CloudNamesTest.java

```diff
@@ -33,33 +34,58 @@ public void isUnique_false() {
 
   @Test
   public void generateUnique_noSuffix() {
-    Assert.assertEquals("FleetCloud", CloudNames.generateUnique("FleetCloud"));
+    Assert.assertEquals("UniqueCloud", CloudNames.generateUnique("UniqueCloud"));
```

Comment:
I guess if `proposedName` checks are removed in, then there is no need to rename the clouds in these tests?